Exercise_B/index.css

```diff
@@ -2,3 +2,33 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+#container {
+    margin: 10px;
+    height: 200px;
+    width: 600px;
+}
+
+#blue {
+    float: left;
+    height: 100px;
+    width: 100px;
+    background-color: blue;
+}
+
+#green {
+    float: right;
+    height: 100px;
+    width: 100px;
+    background-color: green;
+}
+
+
+#red {
+    position: absolute;
+    margin-top: 100px;
+    margin-left: 250px;
+    height: 100px;
+    width: 100px;
+    background-color: red;
+}
```

Comment:
Nice use of margins to move the red box into place, you can improve this code by using the clear float css property.

You did recreate the image though so well done!